sysutils/edid-decode/Portfile

```
@@ -0,0 +1,17 @@
+PortSystem       1.0
```

This Portfile is a good candidate for the `makefile` portgroup.

Portgroups provide default functionality to a Portfile for the way a software project is structured, such as the `golang` or `cargo` (Rust) portgroups.

The `makefile` portgroup will add the right C compiler flags for the given macOS version and platform.

You can add this by deleting the `use_configure no` line below (since the `makefile` PG does this by default), and adding the following:

```suggestion
PortSystem       1.0
PortGroup        makefile 1.0
```

---

Fixed (d73a890).